iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothub/HubTierConnectionAndroidRunner.java

```diff
@@ -15,8 +15,8 @@
 @RunWith(Parameterized.class)
 public class HubTierConnectionAndroidRunner extends HubTierConnectionTests
 {
-    public HubTierConnectionAndroidRunner(DeviceClient client, IotHubClientProtocol protocol, BaseDevice identity, AuthenticationType authenticationType, String publicKeyCert, String privateKey, String x509Thumbprint, boolean useHttpProxy)
```

Comment:
There was no reason to make these x509 details part of the test parameterization, especially since not every test uses them.

Reply:
Most of the files changed in this PR are just doing the same thing in all the android/jvm runner classes